insights/parsers/numeric_user_group_name.py

```diff
@@ -0,0 +1,62 @@
+"""
+NumericUserGroupName - command ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+=========================================================================================
+
+This parser reads the output of ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+and detects whether it is 0 or nonzero.
+
+The grep command returns the number of matches. It matches for user and group names that
+begin with a number. If it returns 0, then there are no such names, otherwise it returns
+the number of matches (names). The parser only detects whether it is 0 or nonzero so as
+to minimize the amount of collected data.
+
+The boolean is available in the ``data`` attribute but it should not be accessed directly
+(encapsulation). The parser provides methods for comparisons with bool and casting to bool.
+
+Sample input::
+
+    3
```

Comment:
I think this is wrong. Sample output of the command is:

```
/etc/passwd:1
/etc/group:1
```

Also you can store how many users and/or groups has name starting with number.